scripts/pipeline/populate_new_database.pl

```diff
@@ -335,8 +335,8 @@ =head2 OLD DATA
 ## Copy all the SpecieTree entries
   copy_all_species_tres($old_dba, $new_dba, $all_default_method_link_species_sets);
 
-## Copy DNA-DNA alignemnts
-  copy_dna_dna_alignements($old_dba, $new_dba, $all_default_method_link_species_sets);
+## Copy DNA-DNA alignments
+  copy_dna_dna_alignments($old_dba, $new_dba, $all_default_method_link_species_sets);
```

Comment:
```suggestion
  copy_dna_to_dna_alignments($old_dba, $new_dba, $all_default_method_link_species_sets);
```
If we change the name like, it is sucky because it is longer, but it makes more sense/is more descriptive and doesn't require the comments at all.